Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine AS builder
+
+WORKDIR /go/src/
+ADD . .
+
+RUN export CGO_ENABLED=0 && go install && go build
+
+FROM scratch AS runtime
+VOLUME /tmp/shhgit
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /go/src/shhgit .
+COPY --from=builder /go/src/config.yaml .
```

Comment:
We can copy this from the repo instead of the builder

Reply:
the config file should not be copy from the builder, but the other ones can be.
Regarding the config.yaml, maybe we should not include it at all and we should pass it when we run the container as a volume

Reply:
Yeah i agree. that's the cleanest way